x-pack/platform/plugins/shared/screenshotting/server/browsers/chromium/templates/index.ts

```diff
@@ -8,12 +8,12 @@
 import { i18n } from '@kbn/i18n';
 import fs from 'fs/promises';
 import path from 'path';
-import Handlebars from 'handlebars';
+import Handlebars, { TemplateDelegate } from '@kbn/handlebars';
 import { assetPath } from '../../../constants';
 
-async function compileTemplate<T>(pathToTemplate: string): Promise<Handlebars.TemplateDelegate<T>> {
+async function compileTemplate<T>(pathToTemplate: string): Promise<TemplateDelegate<T>> {
   const contentsBuffer = await fs.readFile(pathToTemplate);
-  return Handlebars.compile(contentsBuffer.toString());
+  return Handlebars.compileAST(contentsBuffer.toString());
```

Comment:
__question:__ can we limit helpers in this context?
```js
  return Handlebars.compileAST(contentsBuffer.toString(), { knownHelpersOnly: true });
```

and maybe even?
```js
  return Handlebars.compileAST(contentsBuffer.toString(), { knownHelpers: { set all not needed built-in helpers to `false`}, knownHelpersOnly: true });
```

Reply:
Just committed this - 69dd2d7bf34fb6729b5b212da71e85b51a8666f5 - locking down everything but `#if` - and it seems to do everything correctly.